### DISCUSSION TITLE
LTG Summer 2023 - E02

### DISCUSSION THREAD

**Discussion by benjamin-leiding** [ETCE-LAB/teaching-material#21] [Polls]
Poll for E02

**Comment by saikumar-guru** [comment]
![ET_EXERCISE2_SS](https://user-images.githubusercontent.com/105155258/235430475-8697d479-825d-4265-a472-b7b47bcd6652.png)
Sai Kumar Polavarapu, 533308

**Comment by joepaul341** [comment]
[Limits to Growth SS23 Exercise2.pdf](https://github.com/ETCE-LAB/teaching-material/files/11381759/Limits.to.Growth.SS23.Exercise2.pdf)
